### DISCUSSION TITLE
Class file wrong version 61, should be 52

### DISCUSSION THREAD

**Discussion by prani88** [ozlerhakan/poiji#257] [Q&A]
Hello, 
How can I use this package into my project without changing my jdk version

**Comment by ozlerhakan** [comment]
Hi @prani88 , you can use older versions of Poiji. Poiji with the latest version supports Java 17.

**Reply by prani88** [reply]
There are so many versions. Can you please specify which one should be compatible with java 8

**Reply by prani88** [reply]
I tried with older version of 3.1.3 and 2.7.0 ,but @ExcelCellName and other annotations are not working

**Reply by ozlerhakan** [reply]
Hmm could you try 3.1.2

**Reply by prani88** [reply]
Working. Thank you 🙂

**Reply by ozlerhakan** [reply]
you're welcome :). Could you please mark this as answer? Thanks!
